supervisor/homeassistant/core.py

```
@@ -392,7 +397,7 @@ async def _block_till_run(self, version: str) -> None:
                 break
 
         self._error_state = True
-        raise HomeAssistantError()
+        raise HomeAssistantCrashError()
```

This needs an update on line 187 as well

---

Nevermind  `HomeAssistantCrashError(HomeAssistantError)`

---

Line should be 187 catch all exception from this function

---

Yup, which is why I marked it as resolved :)